src/postprocess/apirequests.py

```diff
@@ -117,22 +120,22 @@ def get_users_by_group_names(api, user_group_names):
         return []
 
     response = api.get(
-        "/userGroups",
+        "/users",
         {
             "paging": False,
-            "filter": "name:in:[%s]" % ",".join(user_group_names),
-            "fields": ("id,name," "users[:all,userCredentials[:all,userRoles[id,name]]]"),
+            "filter": "userGroups.name:in:[%s]" % ",".join(user_group_names),
```

Comment:
That change of userGroups->users for user (with filter) is usually a good idea, but note that using filter `in:[...]` prevents user groups from having characters `[],`.

Reply:
Imagine you have a group with name `"Admins, Managers"`

Reply:
Got it, you're right, but in widp we don't have that kind of names for usergroups.
The logical thing would be to use uids, but we would have to talk to nacho before change it because it is intentional to use readable usergroup names instead of uids.

That standard is not followed everywhere... these are the basic actions widp admins should understand.
The alternative is scape that , in the filter but I haven't found a way to escape the comma, maybe you know a trick?

Reply:
Indeed, that filter is typically used with IDs. There is no escaping mechanism, AFAIK, so there is no safe way to filter objects by a collection of names other than getting all of them and filtering them programmatically. 

But as you say, user groups don't typically contain commas or brackets.